src/lib/containers/query_wrapper_plot_nav/TopLevelControls.tsx

```diff
@@ -128,79 +128,81 @@ const TopLevelControls = (
     }
     updateParentState!({ isColumnSelected: isColumnSelectedCopy })
   }
-
+  const showFacetFilter = topLevelControlsState?.showFacetFilter
   return (
-    <h3 className="QueryWrapperPlotNav__title">
-      <div className="QueryWrapperPlotNav__querycount">
-        <QueryCount token={token} name={name} sql={sql} parens={true} />
-      </div>
-      <div className="QueryWrapperPlotNav__actions">
-        {customControls &&
-          customControls.map(customControl => {
+    <div className={`TopLevelControls ${showFacetFilter ? QUERY_FILTERS_EXPANDED_CSS : QUERY_FILTERS_COLLAPSED_CSS}`}>
```

Comment:
Highly recommend you view this diff in "split" view rather than "unified".  It looks like many changes when in "unified" mode, but in reality it just adds a wrapper div with "TopLevelControls" and the expand/collapse filters class.